apps/src/lib/util/firehose.js

```diff
@@ -1,23 +1,18 @@
 /** @file Provides clients to AWS Firehose, whose data is imported into AWS Redshift. */
 
-import AWS from 'aws-sdk';
+import $ from 'jquery';
 
-const IDENTITY_POOL_ID = 'us-east-1:fbfec393-0afb-4682-84d4-59ad04a302f4';
-const AWS_REGION = 'us-east-1';
-
-AWS.config.region = AWS_REGION;
-AWS.config.credentials = new AWS.CognitoIdentityCredentials({
-  IdentityPoolId: IDENTITY_POOL_ID,
-});
-const FIREHOSE = new AWS.Firehose({apiVersion: '2015-08-04'});
+const FIREHOSE_URLS = {
+  "analysis-events" : "https://vqaozk0x32.execute-api.us-east-1.amazonaws.com/prod/FirehoseMicroservice"
+};
 
 /**
  * A barebones client for posting data to an AWS Firehose stream.
  * Usage:
- *   FirehoseClient.putRecord(
+ *   firehoseClient.putRecord(
  *     'analysis-events',
  *     {
- *       created_at: new Date(),             // REQUIRED
+ *       created_at: new Date().toISOString(),             // REQUIRED
```

Comment:
This might not be strictly necessary since we are eventually running the result through JSON.stringify (which should call .toISOString() behind the scenes on Date objects), however it doesn't hurt to be explicit, since the normal toString() function outputs a date format that the Redshift copy command doesn't recognize.